src/ui/component/disciteParts/DisciteModuleActions.html

```diff
@@ -1,9 +1,10 @@
 <div class="discite__action discite__action--module">
 
-
-    {@html `<a href="${cta.url}" ${cta.learningAction.attribute} >`}
-        {$i18n(cta.learningAction.text)}
-    {@html `</a>`}  {cta.learningAction.fileHelper ? cta.learningAction.fileHelper : ''}
+    {#if course.status !== "Archive" }   
```

Comment:
BUG - Service sets the status as `Archived`.

@m24murray could you please help mick to extract this to a method to encapsulate the logic in one place?

```suggestion
    {#if course.isArchived() }   
```

Reply:
This has now been added